lib/config/definitions.ts

```diff
@@ -178,6 +178,14 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'deepExtract',
+    description: `Enable extraction of dependencies using package managers.`,
+    type: 'boolean',
+    default: false,
```

Comment:
If you're not ready to merge a breaking change yet, I'd recommend setting this to `true`, merging this PR as a non-breaking change, and then open another one which is marked as breaking where you change this to `false`.

Lowers the risk of getting merge conflicts if you want to wait and collect many breaking changes into a single release.